### DISCUSSION TITLE
large number of variables manager?

### DISCUSSION THREAD

**Discussion by esba1ley** [snl-dakota/dakota#144] [Q&A]
Dakota community:

Is there some sort of tool that helps analysts manage large numbers of variables across Dakota input files and simulation input deck templates for dprepro?  At JPL in the Guidance and Control Section, we wrote our own, called the "Dakota Analysis Master Parameter Pre-processor (DAMPP) that allows us to use hundreds to thousands of variables for studies by substituting the lists into the proper variable blocks with a matching "id_variables".  Is this something you'd like to see over in the "show and tell"?  It could be a useful addition for other users/organizations...

**Comment by jadamstephens** [comment]
Hi,

I haven't heard of a tool like that before. Please feel free to share it.
